js/sound-player.js

```diff
@@ -38,25 +40,45 @@ function handleVolumeUpdate(event) {
   audio.volume = volume;
 }
 
+// Listen to see which slider is used to raise/lower volume
 volumeSliders.forEach(item => item.addEventListener('input', handleVolumeUpdate));
 
+const forEach = function(array, callback, scope) {
```

Comment:
what issues did we run in to where we needed to implement `forEach` in our app code?